nimbox.nim

```diff
@@ -168,6 +168,24 @@ proc print*[T](_: Nimbox, x, y: int, text: string,
     tbChangeCell(cast[cint](x + i), yInt, c.uint32, fgInt, bgInt)
     i.inc()
 
+proc print*[T](_: Nimbox, x, y: int, text: string,
+               fg: int, bg: int, style: T) =
```

Comment:
Would be nice to use the ``Color`` type from the stdlib ``colors`` module here, sucks that it clashes with the ``Color`` type defined in this module though :/